.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+
+script: "mvn cobertura:cobertura"
```

Comment:
as far as I remember we discussed to use Gradle as build tool.
Think mvn isn't suitable in this case